scripts/mgear/shifter_classic_components/arm_2jnt_02/__init__.py

```diff
@@ -192,6 +194,12 @@ def addObjects(self):
             self.root, self.getName("ik_cns"), self.guide.pos["wrist"])
 
         t = transform.getTransformFromPos(self.guide.pos["wrist"])
+
+        if self.settings["guideOrientWrist"]:
+            t = self.guide.tra["wrist"]
+
+            self.ik_cns.setMatrix(t)
```

Comment:
I've having a hard time matching the ik controller's parent, and have its transform values zeroed out.

Reply:
To be honest. I was planning to use a joint in some point to avoid this situation. The joint has the joint orient values that can reset the orientation while keeping the world align for translation.
Not sure if this will help you. But is another option, maybe easier :)

Reply:
The point of this PR is to make the wrist controllers FK and IK to match with the guide, since currently its matched to world space.

What I'm having a hard time figuring out is where does the IK wrist controllers grandparent get created?